DataFormats/HGCalReco/interface/Trackster.h

```diff
@@ -199,5 +199,7 @@ namespace ticl {
     // trackster ID probabilities
     std::array<float, 8> id_probabilities_;
   };
+
+  typedef std::vector<Trackster> TracksterCollection;
```

Comment:
this was the only line for reco in #32941 
is this PR replacing #32941  or is there more?
From the reco POV, it looks the same now.

Reply:
No, this PR is independent of #32941.
The PR containing the new data member for #32941 has been already merge (#33290).

Reply:
sorry, the update is indeed different; even though in both cases it's a matter of one line for the reco category.